iiblib/iibclient.py

```diff
@@ -504,10 +504,12 @@ def add_bundles(
         post_data = {
             "from_index": index_image,
             "binary_image": binary_image,
-            "bundles": bundles,
             "add_arches": arches,
         }
 
+        if bundles:
```

Comment:
Since we are allowing `bundles` to be not set. Should not be also `bundles` argument optional? 
Or do we want the user to pass empty list on purpose?

Reply:
I think it makes more sense if None is entered by the user specifically. Otherwise, the operation add_bundles would by default not add any bundles.